core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -208,10 +208,16 @@ protected void startMasters(boolean isLeader) {
 
   /**
    * Stops all masters, including block master, fileSystem master and additional masters.
+   *
+   * @param isClose {@code true} if process is exiting
    */
-  protected void stopMasters() {
+  protected void stopMasters(boolean isClose) {
```

Comment:
Will it be better to use `closeMasters()` instead of a boolean flag?

Reply:
Makes sense.